runtime/near-vm-runner/src/imports.rs

```diff
@@ -45,6 +45,7 @@ wrapped_imports! {
     predecessor_account_id<[register_id: u64] -> []>,
     input<[register_id: u64] -> []>,
     block_index<[] -> [u64]>,
+    timestamp<[] -> [u64]>,
```

Comment:
Why not `block_timestamp`?

Reply:
let's change to block_timestamp and can push this.